nginx-auto-config.go

```diff
@@ -95,7 +114,10 @@ func main() {
 		if server.port == 443 {
 			printCautionSSL()
 		}
+		os.Exit(0)
```

Comment:
This is not needed, there's an exit immediately after the block

Reply:
Done